src/type.ts

```diff
@@ -0,0 +1,464 @@
+export interface INetworks {
```

Comment:
Please move these types into the network-stats feature folder since they are only used there